mono/metadata/class.h

```diff
@@ -268,6 +268,12 @@ mono_method_can_access_field (MonoMethod *method, MonoClassField *field);
 MONO_API mono_bool
 mono_method_can_access_method (MonoMethod *method, MonoMethod *called);
 
+MONO_API MonoMethod*
+mono_method_get_method_definition(MonoMethod *method);
+
+MONO_API void
+mono_class_set_allow_gc_aware_layout(mono_bool allow);
```

Comment:
I think it's safe to remove MONO_API here if these aren't in the def file and things work.